### ISSUE TITLE
DEV: LOG: Create the query handler for mapping the data to post data

### ISSUE THREAD

**Issue by cherrera2001** [hashmapinc/Drillflow#318]

**Comment by niaalex** [comment]
In Progress#364

**Comment by niaalex** [comment]
@TessForGithub2 Will review and update later.

**Comment by niaalex** [comment]
In Progress ETA 4/26 with AddToStore

**Comment by TessForGithub2** [comment]
Update 4/26: This is in progress and should be completed today. Will release once JUnit tests are completed.

**Comment by TessForGithub2** [comment]
This card pertains to convertTo1411(ChannelSet 

1. unmarshal incoming JSON String that represents channelSet + channels as a response from DoT
2. now the POJOs are populated from the unmarshal operation, so perform the transformations per mapping document to return a JSONObject ready for transformation to WITSML XML (performed by the caller before returning as XMLOUT)

I have completed the unmarshal; I will test once Sukhe puts in the fix I requested last night so he is sending me more than channels (a JSONArray) but a JSONObject as described above.

I will continue on with mapping & writing JUnit testing against this module. The mapping activity itself is documented in this card, the JUnit testing is in card #402, and all of this is being handled in branch #364.
